x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/math/ETests.java

```diff
@@ -29,14 +29,22 @@ public ETests(@Name("TestCase") Supplier<TestCaseSupplier.TestCase> testCaseSupp
 
     @ParametersFactory
     public static Iterable<Object[]> parameters() {
-        return parameterSuppliersFromTypedData(List.of(new TestCaseSupplier("E Test", List.of(DataType.INTEGER), () -> {
-            return new TestCaseSupplier.TestCase(
-                List.of(new TestCaseSupplier.TypedData(1, DataType.INTEGER, "foo")),
-                "LiteralsEvaluator[lit=2.718281828459045]",
-                DataType.DOUBLE,
-                equalTo(Math.E)
-            );
-        })));
+        return parameterSuppliersFromTypedDataWithDefaultChecks(
+            true,
+            List.of(
+                new TestCaseSupplier(
+                    "E Test",
+                    List.of(DataType.INTEGER),
+                    () -> new TestCaseSupplier.TestCase(
+                        List.of(new TestCaseSupplier.TypedData(1, DataType.INTEGER, "foo")),
```

Comment:
Shouldn't this be the empty list? That looks like what we're doing with `Now` and `Pi`, and makes more sense for a zero parameter constant function.

Reply:
Absolutely! I removed some cases like this in other functions, this one may have slipped. May be related with the CI errors actually 👀